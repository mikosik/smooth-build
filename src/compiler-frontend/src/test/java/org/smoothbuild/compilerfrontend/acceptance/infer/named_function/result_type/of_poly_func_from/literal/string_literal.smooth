myFunc<A>(A a) = "abc";
