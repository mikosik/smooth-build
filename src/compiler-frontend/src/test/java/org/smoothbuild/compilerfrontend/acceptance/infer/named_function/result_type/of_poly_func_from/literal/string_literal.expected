String myFunc<A>(A a)
  = "abc";
